packages/config/src/index.ts

```diff
@@ -48,6 +48,27 @@ class TruffleConfig {
     return pick(options, optionsWhitelist);
   }
 
+  private addDefaultNetworks(): TruffleConfig {
+    const dashboardHost = this.dashboard.host === "0.0.0.0"
+      ? "localhost"
+      : this.dashboard.host;
+
+    const dashboardNetwork = {
+      url: `http://${dashboardHost}:${this.dashboard.port}/rpc`,
+      network_id: "*",
```

Comment:
Should some of these settings just go in `configDefaults.js`? Like the ones that aren't calculated? That is why we need this method, right?

Reply:
Yeah exactly, I don't think we can add the dynamic ones there easily. So since I couldn't have it *all* in the default values I figured it'd be cleaner to move it all out of that file, so at least it's in the same place

Would you suggest that we put some values in the defaults, like:

```js
  networks: {
    dashboard: {
      network_id: "*"
      ...
    }
  }
```

And then add "networks" to the deepCopy list so that they get merged with the user overrides?